### ISSUE TITLE
System status and monitoring

### ISSUE THREAD

**Issue by docsteveharris** [HYLODE/HyUi#133]
who will be monitoring
where are the logs
logs are then monitored
and then if logs are dead it does some sort of check

**Comment by docsteveharris** [comment]
First draft sketched out
Suggest we add https://prometheus.io/docs/guides/cadvisor/ before closing this
Need to scope this to the minimum necessary i.e. 'what's working'
And decide if this is part of HyUI or HySys. 
I suspect the latter but we will need a guide to allow users to instrument the applications sharing data

**Comment by docsteveharris** [comment]
what would be simple, do-able, and worthwhile
for discussion and really just to decide what's worth doing in the last few days

**Comment by harryjmoss** [comment]
Agreed that minimum monitoring on the HySys side is that recent data is coming through into HyFlow. 

@jongillham adding a quick check for this and wrapping in a FastAPI endpoint
